compiler/src/iree/compiler/Modules/IO/Parameters/Conversion/StreamToParams/Patterns.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_COMPILER_MODULES_IO_PARAMETERS_CONVERSION_STREAMTOPARAMS_PATTERNS_H_
+#define IREE_COMPILER_MODULES_IO_PARAMETERS_CONVERSION_STREAMTOPARAMS_PATTERNS_H_
+
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir::iree_compiler {
+
+// Populates conversion patterns for stream->HAL (loader).
+void populateStreamToIOParametersPatterns(MLIRContext *context,
```

Comment:
Update comment (or just remove, function name is self-explanatory)

```suggestion
void populateStreamToIOParametersPatterns(MLIRContext *context,
```